### PR TITLE
109 page not found

### PR DESCRIPTION
**Pull request by BrookeVerse**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { Route, Routes } from "react-router-dom";
-import Home from "./pages/Home/Home";
```
Comment (OmarKarame) [inline src/App.jsx:2]: Line 2: Return the Home import.

File: src/App.jsx [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+import PageNotFound from "./components/PageNotFound/PageNotFound";
```
Comment (OmarKarame) [inline src/App.jsx:2]: Line 3: Remove import

File: src/App.jsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 import "./styles/main.scss";
 
 const App = () => {
   return (
     <Routes>
-      <Route path="*" element={<Home />} />
```
Comment (OmarKarame) [inline src/App.jsx:8]: Line 8: Return Route path for home (<Route path="*" element={<Home />} />)

File: src/App.jsx [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
+      <Route path="*" element={< PageNotFound/>} />
```
Comment (OmarKarame) [inline src/App.jsx:8]: Line 8: Remove Route

File: src/components/PageNotFound/PageNotFound.test.jsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  });
+
+  it("Should render an image", () => {
+    render(<PageNotFound />, { wrapper: BrowserRouter });
+
+    const image = screen.getByRole("img");
+    expect(image).toBeInTheDocument();
+  });
+
+  it("Should render an button", () => {
```
Comment (OmarKarame) [inline src/components/PageNotFound/PageNotFound.test.jsx:19]: Line 19: Change "an button" to "a button"

**Review by OmarKarame — CHANGES_REQUESTED** [review]
Please check comments and remember to return Home to its original state.

File: src/App.jsx (hunk 1 above)
Comment (BrookeVerse) [reply src/App.jsx:2]: Made Change

File: src/App.jsx (hunk 2 above)
Comment (BrookeVerse) [reply src/App.jsx]: Made Change

File: src/App.jsx (hunk 4 above)
Comment (BrookeVerse) [reply src/App.jsx]: Made Change

**Review by Connz17 — APPROVED** [review]

**Pull request merged**